mlflow/ml-package-versions.yml

```diff
@@ -656,12 +656,14 @@ langchain:
       if [[ $LANGCHAIN_NEW = "True" ]]; then
         pytest tests/langchain/test_langchain_tracer.py
       fi
+      pytest tests/langchain/test_langchain_output_parsers.py
```

Comment:
can we avoid this by reorganize the folder/file structure in `tests/langchain`?

Reply:
> can we avoid this by reorganize the folder/file structure in tests/langchain?

You mean putting those tests files under subdirectory like `langchain/model/test_xxx` right? I thought about that, but probably it doesn't work as permanent solution either, because there is no forcing-function for ppl to put a new test file under that subdirectory😔

Reply:
oh maybe a better solution is to use `--ignore` flag to run all tests but autolog?
```
pytest tests/langchain --ignore tests/langchain/test_langchain_autolog.py
```
I found we actually do this for transformers.

Reply:
> because there is no forcing-function for ppl to put a new test file under that subdirectory😔

What if we add this?

```python
# tests/langchain/__init__.py
from pathlib import Path

if any(f.name.startswith("test_") for f in Path(__file__).parent.glob("*.py")):
    raise Exception("DO NOT ADD TEST FILES TO THIS DIRECTORY!")
```

Reply:
hmmm that works too, but I feel it's a bit confusing for contributors - not super clear why it needs to be as such and which directory to choose.